frontend/src/components/WaiverComponent/WaiverPage.jsx

```diff
@@ -114,11 +147,72 @@ export default function WaiverPage() {
           <WaiverFormRightWrapper>
             <AgreementSection>
               <AgreementText>
-                Check here to indicate that you have read and agree to the terms
+                Click here to indicate that you have read and agree to the terms
                 of the City Farm SLO Volunteer Agreement
               </AgreementText>
-              <CheckBox />
+              {/* <CheckBox /> */}
+              {/* Checkbox for City Farm SLO Volunteer Agreement */}
+              <Radio
+                sx={{
+                  "&:hover": {
+                    backgroundColor: "transparent",
+                    "@media (hover: none)": {
+                      backgroundColor: "transparent",
+                    },
+                  },
+                  paddingRight: "485px",
+                  color: "white",
+                  "&.Mui-checked": {
+                    color: "white",
+                    transitionDuration: "0s !important",
+                  },
+                }}
+                checked={checked === true}
+                onClick={handleChange1}
+                name="radio-buttons"
+              />
+              <AgreementText>Print your name</AgreementText>
+              {/* Added text field here */}
+              <TextField
+                id="filled-basic"
+                variant="filled"
+                className={classes.root}
+              />
             </AgreementSection>
+            {/* Conditional rendering for whether Volunteer isAdult or not */}
+            {isAdult === true ? (
```

Comment:
This is a really small nitpick, but since isAdult is a boolean, the `=== true` part of this code is redundant.